nekoyume/Assets/_Scripts/Game/Stage.cs

```diff
@@ -470,7 +471,14 @@ private IEnumerator CoRankingBattleEnter(BattleLog log)
 
         private IEnumerator CoStageEnd(BattleLog log)
         {
+            AvatarState avatarState = null;
+            ActionRenderHandler.Instance.Pending = false;
             _onEnterToStageEnd.OnNext(this);
+            yield return ActionRenderHandler.Instance
+                .AvatarStateForBattleResultSubject
+                .First()
+                .Subscribe(value => avatarState = value);
+
```

Comment:
```suggestion
            AvatarState avatarState = null;
            yield return ActionRenderHandler.Instance
                .AvatarStateForBattleResultSubject
                .DoOnSubscribe(() =>
                {
                    ActionRenderHandler.Instance.Pending = false;
                    _onEnterToStageEnd.OnNext(this);
                })
                .First()
                .Subscribe(value => avatarState = value);
```

Reply:
`ActionRenderHandler.Instance.AvatarStateForBattleResultSubject` is pre-instantiated subject. Are they also invoked on calling `DoOnSubscribe`?